connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -534,6 +536,36 @@ class SparkConnectPlanner(session: SparkSession) {
     }
   }
 
+  /**
+   * Translates a LambdaFunction from proto to the Catalyst expression.
+   */
+  private def transformLamdaFunction(lambda: proto.Expression.LambdaFunction): Expression = {
+    if (lambda.getArgumentsCount == 0) {
```

Comment:
Do we get an analysis exception if more than three arguments are submitted?

Reply:
There is a validation in the client side already but I guess we need that in the server side as well?